### PR TITLE
Update template for sagebridge

### PR DESCRIPTION
**Pull request by zaro0508**
* add EC2KeyName parameter
* add SSLCertArn parameter
* change references from "museborn" to "sagebridge"
* identify secret parameters as "SECRET"

### CODE REVIEW

File: BridgePF-env.template [hunk 1]
```diff
@@ -923,12 +927,12 @@
           },
           {
             "Namespace" : "aws:elasticbeanstalk:application:environment",
-            "OptionName" : "AWS_BRIDGE_HEALTHCODE_KEY",
```
Comment (consolecowboy) [inline BridgePF-env.template]: This change will break the dist/.ebextensions/01_bridge.cf at line 30, but you can update that file to match whatever you want to use as long as you update this file to match.

File: BridgePF-env.template [hunk 2]
```diff
@@ -923,12 +927,12 @@
...
+            "OptionName" : "BRIDGE_HEALTHCODE_KEY",
             "Value" : { "Ref" : "BridgeHealthcodeKey" }
           },
           {
             "Namespace" : "aws:elasticbeanstalk:application:environment",
-            "OptionName" : "AWS_BRIDGE_HEALTHCODE_REDIS_KEY",
```
Comment (consolecowboy) [inline BridgePF-env.template]: This change will break the dist/.ebextensions/01_bridge.cf at line 31, but you can update that file to match whatever you want to use as long as you update this file to match.

Comment (zaro0508) [inline BridgePF-env.template]: 01_bridge.cf was updated to use BRIDGE_HEALTHCODE_KEY and BRIDGE_HEALTHCODE_REDIS_KEY in change https://github.com/Sage-Bionetworks/BridgePF/pull/1439

**Comment by DwayneJengSage** [comment]
As per email conversation, we won't be using this CloudFormation template.

**Pull request closed without merging**